src/main/resources/mybatis/mappers/board/boardMapper.xml

```diff
@@ -3,7 +3,8 @@
   "http://mybatis.org/dtd/mybatis-3-mapper.dtd">
 <mapper namespace="com.mide.gangsaeng.board.BoardMybatisMapper">
   <insert id="write" parameterType="com.mide.gangsaeng.board.Board" useGeneratedKeys="true" keyProperty="id">
-    INSERT INTO boards (title, content, user_id) values (#{title}, #{content}, #{userId});
+    INSERT INTO boards (title, content, user_id, created_at, updated_at)
+    values (#{title}, #{content}, #{userId}, #{createdAt}, #{updatedAt});
```

Comment:
Can you print the actual query when application insert data?

Reply:
<img width="1255" alt="image" src="https://github.com/user-attachments/assets/c08b8602-70e0-4c3e-bac0-cc9ba09da24b">